### PR TITLE
Extended Help feedback

### PR DESCRIPTION
**Pull request by siddhant1**
Added more features as of #237

### CODE REVIEW

**Comment by siddhant1** [comment]
Added `?` on top to make help and feedback section more accessible since no one will know they have to press cntrl+shift+?

**Comment by domoritz** [comment]
Do you have a screenshot?

**Comment by siddhant1** [comment]
image

**Comment by siddhant1** [comment]
Added that `?` above too

**Comment by siddhant1** [comment]
image

**Comment by domoritz** [comment]
The layout in the main view looks off, doesn't it?

**Comment by siddhant1** [comment]
Any particular styles you have in mind? I know it's pretty basic

**Comment by domoritz** [comment]
I was just looking at https://github.com/vega/editor/pull/260#issuecomment-471586032. The rest is great.

**Comment by siddhant1** [comment]
Cool , pushing

**Comment by siddhant1** [comment]
image
Does it look better now?

File: src/components/help-modal/index.tsx [hunk 1]
```diff
@@ -21,24 +21,32 @@ const HelpModal = () => {
...
       </ul>
       <a href="https://github.com/vega/editor/issues/new" target="_blank" className="report-button">
         <AlertCircle className="header-icon" />
         Report a Bug{' '}
       </a>
-      <br />
       <h2>Have a feedback? Let us know</h2>
-      <a href="https://bit.ly/join-vega-slack" target="_blank">
+      <a href="https://bit.ly/join-vega-slack" target="_blank" className="link-to-docs">
+        <img src="/images/utils/slack-logo-icon.png" className="doc-image" />
```
Comment (domoritz) [inline src/components/help-modal/index.tsx]: Please use https://feathericons.com/ like we do in other places (using react feather).

File: src/components/help-modal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      </a>
+      <h2>References</h2>
+      <a href="https://vega.github.io/vega-lite/" target="_blank" className="link-to-docs">
+        Vega-lite Docs
+      </a>
+      <a href="https://vega.github.io/vega/" target="_blank" className="link-to-docs">
+        Vega Docs
+      </a>
+      <h2>Tips</h2>
+      1. You can access and debug vega view using your browsers developer console
```
Comment (domoritz) [inline src/components/help-modal/index.tsx]: Add something about `VEGA_DEUG` variable and how to use it.

Comment (domoritz) [reply src/components/help-modal/index.tsx]: Also capitalize Vega.

Comment (siddhant1) [reply src/components/help-modal/index.tsx]: Sure

**Comment by domoritz** [comment]
The alignment of text and icon in https://github.com/vega/editor/pull/260#issuecomment-471639765 is wrong.

**Comment by siddhant1** [comment]
image

**Comment by algomaster99** [comment]
@siddhant1 @domoritz 
https://github.com/vega/editor/commit/78b00d274821a3411ed7fd0c3a50e4e89f5cc3e5#diff-f73b15975f8fc5805ff859f69034f539R4
I have written a comment here. Please see. Thank you!

**Pull request merged**